src/System/Process/Typed.hs

```diff
@@ -891,7 +890,7 @@ readProcessInterleaved_
   => ProcessConfig stdin stdoutIgnored stderrIgnored
   -> m L.ByteString
 readProcessInterleaved_ pc =
-    liftIO $ do
+    liftIO $
```

Comment:
Removing the `do` is "meh" IMO but it matches the style in the function above this one.